src/librustdoc/config.rs

```diff
@@ -415,6 +435,7 @@ impl Options {
             }
             return Err(0);
         }
+        let (_lint_opts, _describe_lints, _lint_cap) = get_cmd_lint_options(matches, error_format);
```

Comment:
Why are these ignored?

Reply:
Do you remember by any @inashivb ?

Reply:
I _think_ this is leftover from initial mentoring instructions when I was trying it out and can be removed..